packages/sitecore-jss-cli/src/process-env.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path';
+import * as dotenv from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
+
+/**
+ * @param {string} dir The directory containing the .env files to load.
+ */
+export default function processEnv(dir: string) {
+  // replicate Next.js handling/behavior
+  // https://github.com/vercel/next.js/blob/v10.0.5/packages/next-env/index.ts#L80-L90
+  const mode = process.env.NODE_ENV;
```

Comment:
Is the NODE_ENV prop always guaranteed to have a value or do we need to check that before interpolating it?

Reply:
Good call will fix

Reply:
done

Reply:
Based on follow up discussion, now excluding mode .env's if there is not a NODE_ENV.